paper-tabs.html

```diff
@@ -377,6 +378,39 @@
         }, 10);
       },
 
+      _onIronItemsChanged: function(event) {
```

Comment:
Added. Also, I don't think I need to remove the listener above since I think they don't duplicate in the same way that the properties block does.